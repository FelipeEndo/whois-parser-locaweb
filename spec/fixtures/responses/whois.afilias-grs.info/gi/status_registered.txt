Domain Name: SAPPHIRE.GI
Registry Domain ID: D105800000000068296-AGRS
Registrar WHOIS Server:
Registrar URL:
Updated Date: 2017-12-05T13:07:12Z
Creation Date: 2004-12-20T13:34:34Z
Registry Expiry Date: 2022-12-20T13:34:34Z
Registrar Registration Expiration Date:
Registrar: GibNet Registrar
Registrar IANA ID: 119
Registrar Abuse Contact Email:
Registrar Abuse Contact Phone:
Reseller:
Domain Status: ok https://icann.org/epp#ok
Registrant Organization: Broadband Gibraltar Limited
Registrant State/Province: Not Available
Registrant Country: GI
Name Server: NS1-A.SAPPHIRE.GI
Name Server: NS2-A.SAPPHIRE.GI
DNSSEC: unsigned

>>> Last update of WHOIS database: 2021-11-09T15:08:05Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy.
The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
