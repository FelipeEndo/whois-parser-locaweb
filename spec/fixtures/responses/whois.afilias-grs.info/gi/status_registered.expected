#disclaimer
  %s == "Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy. The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."


#domain
  %s == "sapphire.gi"

#domain_id
  %s == "D105800000000068296-AGRS"


#status
  %s == ["ok https://icann.org/epp#ok"]

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2004-12-20 13:34:34 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-12-05 13:07:12 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2022-12-20 13:34:34 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "119"
  %s.name         == "GibNet Registrar"
  %s.url          == ""

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id           == nil
  %s[0].name         == nil
  %s[0].organization == "Broadband Gibraltar Limited"
  %s[0].address      == nil
  %s[0].city         == nil
  %s[0].zip          == nil
  %s[0].state        == "Not Available"
  %s[0].country_code == "GI"
  %s[0].phone        == nil
  %s[0].fax          == nil
  %s[0].email        == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{0}

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{0}


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1-a.sapphire.gi"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2-a.sapphire.gi"
